superset-frontend/package.json

```diff
@@ -118,6 +108,7 @@
     "dnd-core": "^2.6.0",
     "dom-to-image": "^2.6.0",
     "geolib": "^2.0.24",
+    "global-box": "^1.2.0",
```

Comment:
A new `peerDependency` from the updated `encodable`, which is a dependency of `preset-chart-xy`.